build/appveyor/before_build.bat

```diff
@@ -1,13 +1,13 @@
 :: set platform-dependent variables
 IF "%PLATFORM%" == "x64" (
-  SET "QTURL=https://utils.musescore.org.s3.amazonaws.com/qt598_msvc2017_64.7z"

```

Comment:
It's not really necessary now because we've moved all the CI into GitHub Actions

You need to rebase your branch on the latest 3.x

Reply:
Thanks. Done.